bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx68.html

```diff
@@ -0,0 +1,85 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/whatsnew_67" %}
+
+{% extends "firefox/whatsnew/whatsnew-fx67.html" %}
+
+{% block fxa_form %}
+  {{ fxa_email_form(
+      entrypoint='mozilla.org-whatsnew67',
+      button_class='mzp-c-button mzp-t-product mzp-t-small',
+      utm_source='whatsnew',
```

Comment:
other wnp's seem to define the number of the wnp in `utm_source`, like `utm_source='whatsnew6705'`
is this the expected tag for this template?

Reply:
Will update